TodoApp/Tests/TodoApp.Services.Tests/packages.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Castle.Core" version="4.2.0" targetFramework="net472" />
+  <package id="NSubstitute" version="3.1.0" targetFramework="net472" />
+  <package id="NUnit" version="3.10.1" targetFramework="net472" />
+  <package id="NUnit3TestAdapter" version="3.10.0" targetFramework="net472" />
```

Comment:
Technically, you only need the `NUnit3TestAdapter` reference to be present once in your solution (I believe that once it is downloaded as a NuGet and referenced in at least one project of a solution, the discovery of tests will succeed in all projects). You have one more advantage: You have an assembly that is referenced in all test projects - and no other projects - that makes perfect closure (as in set's topology) for such references…